ocs_ci/helpers/disruption_helpers.py

```diff
@@ -153,12 +191,17 @@ def check_new_pid(self, node_name=None):
             node_name (str): Name of node in which the resource daemon is running
 
         """
+        kubeconfig_parameter = (
+            f"--kubeconfig {self.cluster_kubeconfig} "
+            if self.cluster_kubeconfig
+            else ""
+        )
         node_name = node_name or self.resource_obj[0].pod_data.get("spec").get(
             "nodeName"
         )
         awk_print = "'{print $1}'"
         pid_cmd = (
-            f"oc debug node/{node_name} -- chroot /host ps ax | grep"
+            f"oc {kubeconfig_parameter}debug node/{node_name} -- chroot /host ps ax | grep"
```

Comment:
we probably want to use `self.kubeconfig_parameter()` here.

Reply:
Yes, done.